app/services/void_participant_declaration.rb

```diff
@@ -14,6 +14,8 @@ def call
 
     NPQ::VoidParticipantOutcome.new(participant_declaration).call
```

Comment:
I think we need some form of transaction here?

Reply:
Moved the check into the other transaction based actions